tests/engine/render/textures/testRecursive.js

```diff
@@ -0,0 +1,14 @@
+// This is an automatically generated file, created by auto-tester on May 24 2018, 13:34
+
+user = "highfidelity/"
+repository = "hifi_tests/"
+branch = "master/"
```

Comment:
The semicolons at the end of JS lines are optional, right? But they are preferred?